src/__tests__/admin/[entity]/config/crud__toggling.spec.tsx

```diff
@@ -92,17 +92,20 @@ describe("pages/admin/[entity]/config/crud", () => {
 
       const currentTab = screen.getByRole("tabpanel");
 
-      if (tab !== "Delete") {
+      if (tab !== "Delete" && tab !== "Update") {
```

Comment:
@sam1330 
These changes will make the test pass, not actually document the implementation. I see that the changes are breaking the test.
I suggest we reverse the order of the cases so that `details` don't interfere with `update`
so instead of 
```js
    { tab: "Details" },
    { tab: "Create" },
    { tab: "Update" },
    { tab: "Delete" },
````

we will now have 
```
 { tab: "Update" },
    { tab: "Details" },
    { tab: "Create" },
    { tab: "Delete" },
```
this way the update comes before the details and it will no longer interfere with the update and we will not have to touch the tests

WDYT?